### DISCUSSION TITLE
Overlays Meeting (2024-09-10)

### DISCUSSION THREAD

**Discussion by lornajane** [OAI/Overlay-Specification#52] [Meeting Notes]
All meetings are subject to the OpenAPI code of conduct. Please add your agenda topics if you have them!

**Comment by ralfhandl** [comment]
Release 1.0
* #50 
* #47
* #49
* Which of [these issues](https://github.com/OAI/Overlay-Specification/issues?q=is%3Aopen+is%3Aissue+no%3Amilestone) are relevant for Release 1.0?

Housekeeping
* #51 
* #53

**Comment by lornajane** [comment]
Thanks @ralfhandl for the agenda! I'll capture some of my notes from the session as comments here so we can discuss further as needed.

**Comment by lornajane** [comment]
An Overlay can be referred to as such since it is a noun (in comparison to an OpenAPI description which describes something, description is the noun). We will therefore use Overlay as a standalone word to mean some JSON or YAML that can be applied to an OpenAPI description. Overlay description is not in use as a term. Overlay Document could make sense to refer to a file with an Overlay in it, but is not formally defined.

**Comment by lornajane** [comment]
The Intro was updated to make better use of the term Overlay, and the Overlay term was edited in the specification, previously we referred to an "Overlay Document".

**Comment by lornajane** [comment]
Regarding how to refer to updates that target either objects or arrays, we updated the wording for that section again and removed the duplicate reference to how to replace an array since this is covered in the following paragraph. An example is also already included in the specification document.

**Comment by lornajane** [comment]
We reviewed the list of open issues and agreed that we don't have anything else that should be added to the 1.0 milestone. Some of the older existing issues can be revisited and closed around the time of the specification release.

**Comment by lornajane** [comment]
We believe that we are close to having the 1.0 release ready; this will then be open for a period to solicit feedback from current implementors before we publish. We also noted that everyone active in this project is also active in OpenAPI itself, which also has imminent patch releases coming up which will need our attention.
